src/routes.js

```diff
@@ -44,4 +45,16 @@ router.post('/coffeegrams', async ctx => {
   ctx.redirect(`/users/${ctx.state.currentUser.username}`);
 });
 
+router.post('/users/:username/bio', async ctx => {
```

Comment:
That's very interesting...what happens when other editable fields are introduced to the usergram view?